consensus/ethash/ethash.go

```diff
@@ -43,6 +43,11 @@ import (
 	"github.com/hashicorp/golang-lru/simplelru"
 )
 
+const (
+	algorithmName    = "ethash" // PoW algorithm name
+	algorithmVersion = "1.0.0"  // Algorithm revision information
+)
```

Comment:
I think one variable is enough. I don't think we'll ever want to track versions and algos separately, so might as well use a single string.